library/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/recorder/mapper/BaseWireframeMapper.kt

```diff
@@ -0,0 +1,29 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.sessionreplay.recorder.mapper
+
+import android.view.View
+import com.datadog.android.sessionreplay.model.MobileSegment
+import java.util.Locale
+
+internal abstract class BaseWireframeMapper<T : View, S : MobileSegment.Wireframe> :
+    WireframeMapper<T, S> {
+
+    protected fun resolveViewId(view: View): Long {
+        return if (view.id != View.NO_ID) view.id.toLong() else view.hashCode().toLong()
```

Comment:
Hmmm you are right...it might be a problem due the the `view.hashcode` maybe I should use the reference address there instead ? What do you think ? It can create problems if it happens as it might override other view properties in the mutation computation.